src/redist/targets/GenerateLayout.targets

```diff
@@ -288,11 +266,19 @@
       </BundledInstallerComponent>
       <ComponentToDownload Include="@(BundledLayoutComponent);@(BundledInstallerComponent)">
         <ShouldDownload Condition="!Exists('%(DownloadDestination)')">true</ShouldDownload>
+
+        <!--
+          Replaces the host portion of 'BaseUrl' with 'PrivateBaseUrl'.
+        -->
+        <PrivateBaseUrl Condition="'$(InternalBuild)' == 'true'">$(MSRCBaseURL)$([System.String]::new('%(BaseUrl)').Substring($([System.String]::new('%(BaseUrl)').IndexOf('/', 8))))</PrivateBaseUrl>
```

Comment:
Unclear why `8` is chosen. It seems like this could just be a `Replace`.

Reply:
Yeah. Sorry about that. I had a newer version of this file where I actually use a Replace and I don't have the <Error.../> thing below.